test/contrib/external_program_test.py

```diff
@@ -104,6 +104,29 @@ def test_handle_failed_job(self, proc, file, logger):
         else:
             self.fail('Should have thrown ExternalProgramRunError')
 
+    @patch('luigi.contrib.external_program.tempfile.TemporaryFile')
+    @patch('luigi.contrib.external_program.subprocess.Popen')
+    @patch.dict('os.environ', {'SECRET_TOKEN': '1l0v3ca75'}, clear=True)
+    def test_print_environment_on_failure_if_debug_environment(self, proc, file):
```

Comment:
Can you add one more test to verify that the default behavior is still print the environment variables

Reply:
See a185763. But what is wrong with the codecov checks?